src/libImaging/Unpack.c

```diff
@@ -1531,16 +1666,24 @@ static struct {
     ImagingShuffler unpack;
 } unpackers[] = {
 
-    /* raw mode syntax is "<mode>;<bits><flags>" where "bits" defaults
-       depending on mode (1 for "1", 8 for "P" and "L", etc), and
-       "flags" should be given in alphabetical order.  if both bits
-       and flags have their default values, the ; should be left out */
-
-    /* flags: "I" inverted data; "R" reversed bit order; "B" big
-       endian byte order (default is little endian); "L" line
-       interleave, "S" signed, "F" floating point */
-
-    /* exception: rawmodes "I" and "F" are always native endian byte order */
+    // The rawmode syntax is "<mode>;<bits><flags>".
+    // "bits" defaults depending on mode (1 for "1", 8 for "P" and "L", etc.).
+    // "flags" should be given in alphabetical order.
+    // If both bits and flags have their default values, the ; should be left out.
+
+    // Flags:
+    // "B" big endian byte order (default is little endian)
+    // "F" floating point
+    // "I" inverted data
```

Comment:
```suggestion
    // "I" inverted data
    // "F" floating point
```
With this minor change, the flags become alphabetical